solution/deps.yaml

```diff
@@ -91,9 +92,9 @@ vault:
   tag: 8.7.2
   envsubst: VAULT_TAG
 zenko-operator:
-  sourceRegistry: registry.scality.com/zenko-operator
+  sourceRegistry: registry.scality.com/sf-eng
   image: zenko-operator
-  tag: 1.5.7
+  tag: 6b5f0255b30f1db2d6b3968791359846d18d37f3
```

Comment:
Just a note not to forget to change this.
Also maybe you will want to bump Zenko version!